src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -422,6 +422,18 @@ public override void ContinueAsNew(object input, bool preserveUnprocessedEvents)
         {
             this.ThrowIfInvalidAccess();
 
+            // These check can be removed once the storage provider supports extended timers.

```

Comment:
Debatably I can remove this comment both here and in the Timers section, as it is highly unlikely that we will ever update the Storage SDK in Durable 1.0.